src/cls/IPM/Main.cls

```diff
@@ -1857,12 +1857,13 @@ ClassMethod Load(ByRef pCommandInfo) [ Internal ]
 {
 	Set tDirectoryName = $Get(pCommandInfo("parameters","path"))
 	Merge tParams = pCommandInfo("data")
-  Set tParams("cmd") = "load"
-  if $Extract(tDirectoryName,1,4)="http" {
+	Set tParams("cmd") = "load"
+	Do ##class(%Net.URLParser).Decompose(tDirectoryName, .tComponents)
+	if $G(tComponents("scheme")) {
```

Comment:
This isn't quite right - strings aren't truthy. I would recommend:
if $Get(tComponents("scheme")) [ "http"